### DISCUSSION TITLE
Revised version of the FreeCAD icon

### DISCUSSION THREAD

**Discussion by sesvena** [FreeCAD/FreeCAD#10270] [Discussions that will turn in to issues but need refining]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Full version info

```shell
[code]
OS: Windows 11 build 22621
Word size of FreeCAD: 64-bit
Version: 0.21.0.33668 +7 (Git)
Build type: Release
Branch: (HEAD detached at 0.21)
Hash: 41b058e2087de60dc8fef3d3e68c7d0129e13abf
Python 3.8.10, Qt 5.15.2, Coin 4.0.1, Vtk 8.2.0, OCC 7.6.3
Locale: German/Germany (de_DE)
[/code]
```

### Subproject(s) affected?

None

### Problem description

Hello,

attached is a revised version of the FreeCAD icon.
The individual icons are included in SVG format and are pixel-perfectly adapted to the respective size.
If you are interested in my design and need more sizes, please contact me.

Greeting Roland

### Anything else?

![FreeCAD_256](https://github.com/FreeCAD/FreeCAD/assets/98457567/1a2b44cd-3402-4ff2-b76c-0239f57277a5)
![FreeCAD_16](https://github.com/FreeCAD/FreeCAD/assets/98457567/ff82e337-76b0-419d-a706-19c489301561)
![FreeCAD_24](https://github.com/FreeCAD/FreeCAD/assets/98457567/bc597dfb-7db2-4828-9f3c-913b6531af72)
![FreeCAD_32](https://github.com/FreeCAD/FreeCAD/assets/98457567/5fe90231-9f7b-4a7a-85ed-2122ba0d4e4b)
![FreeCAD_48](https://github.com/FreeCAD/FreeCAD/assets/98457567/d61a1314-9fcc-4693-8e27-af8b8dec5d4c)
[FreeCAD-Icon.zip](https://github.com/FreeCAD/FreeCAD/files/12380396/FreeCAD-Icon.zip)

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by chennes** [comment]
This isn't really a problem report, so I've migrated to discussions. If you can articulate the _need_ for a new icon, that's a problem report.

**Reply by luzpaz** [reply]
Not sure this is a good idea to start a discussion when there is already an ongoing one in the forum...right?

**Reply by ghost** [reply]
Not sure, GitHub discussion itself is a good idea, since there is a forum already.

Why use a second (or third) platform for discussion, it gets tangle and therefore ambiguous. Also FreeCAD have no **real** admins on its own discussions(platform). Also some people will get blocked soon (like me) or blocked because their IP.

And beyond that, it have a really s***ty usage (even mailing lists are more user friendlier).

**Reply by sesvena** [reply]
@chennes
The problem with the current icon is the display quality at small sizes.
But these are the ones most often used (file explorer, desktop, icon bar, etc.).
My goal was to improve the display on small icons, both on a light and a dark background.

@luzpaz, @grubertechnic
Where in the forum can I find this discussion?

**Reply by luzpaz** [reply]
@sesvena This is the logo thread: https://forum.freecad.org/viewtopic.php?t=50640

@grubertechnic this is a 'growing pain'. An awkward stage of growing beyond the forum. Not sure what you mean by blocked.

**Reply by ghost** [reply]
@luzpaz: there is already a thread in the forum about that, see https://forum.freecad.org/viewtopic.php?t=80095. Since it is bogus (even in its technical form), i will also not sign it, i never let me extort. And on i am not conspirator. Anyway, i will not write anything about it again here.
